provisioning/resources/configs/snowplow-es-loader-bad.hocon

```diff
@@ -0,0 +1,149 @@
+#
+# If both are set to "iam", use AWS IAM Roles to provision credentials.
+#
+# If both are set to "env", use environment variables AWS_ACCESS_KEY_ID and AWS_SECRET_ACCESS_KEY
+aws {
+  accessKey: ""
+  secretKey: ""
+}
+
+# config for NSQ
+nsq {
+  # Channel name for NSQ source
+  channelName: "ESLoaderChannelBad"
+     
+  # Host name for NSQ tools
+  host: "127.0.0.1"
+
+  # Port for nsqd
+  port: 4150
+
+  # Port for nsqlookupd
+  lookupPort: 4161
+}
+
+kinesis {
+  # "LATEST": most recent data.
+  # "TRIM_HORIZON": oldest available data.
+  # "AT_TIMESTAMP": Start from the record at or after the specified timestamp
+  # Note: This only affects the first run of this application on a stream.
+  initialPosition: "TRIM_HORIZON"
+
+  # Need to be specified when initial-position is "AT_TIMESTAMP".
+  # Timestamp format need to be in "yyyy-MM-ddTHH:mm:ssZ".
+  # Ex: "2017-05-17T10:00:00Z"
+  # Note: Time need to specified in UTC.
+  initialTimestamp: ""
+
+  # Maximum number of records to get from Kinesis per call to GetRecords
+  maxRecords: 1000
+
+  # Region where the Kinesis stream is located
+  region: ""
+
+  # "appName" is used for a DynamoDB table to maintain stream state.
+  # You can set it automatically using: "SnowplowElasticsearchSink-${sink.kinesis.in.stream-name}"
+  appName: ""
+}
+
+# Common configuration section for all stream sources
+streams {
+  streamNameIn: "BadEnrichedEvents"
+  
+  # Stream for enriched events which are rejected by Elasticsearch
+  streamNameOut: "BadElasticsearchEvents"
+
+  # Events are accumulated in a buffer before being sent to Elasticsearch.
+  # Note: Buffering is not supported by NSQ; will be ignored
+  # The buffer is emptied whenever:
+  # - the combined size of the stored records exceeds byteLimit or
+  # - the number of stored records exceeds recordLimit or
+  # - the time in milliseconds since it was last emptied exceeds timeLimit
+  buffer {
+    byteLimit: 5242880
+    recordLimit: 1
+    timeLimit: 60000
+  }
+}
+
+elasticsearch {
+
+  # Events are indexed using an Elasticsearch Client
+  # - endpoint: the cluster endpoint
+  # - port: the port the cluster can be accessed on
+  #   - for http this is usually 9200
+  #   - for transport this is usually 9300
+  # - max-timeout: the maximum attempt time before a client restart
+  # - ssl: if using the http client, whether to use ssl or not
+  client {
+    endpoint: "localhost"
+    port: 9200
+    maxTimeout: "10000"
+    ssl: false
+  }
+
+  # When using the AWS ES service
+  # - signing: if using the http client and the AWS ES service you can sign your requests
+  #    http://docs.aws.amazon.com/general/latest/gr/signing_aws_api_requests.html
+  # - region where the AWS ES service is located
+  aws {
+    signing: false
+    region: ""
+  }
+
+  # index: the Elasticsearch index name
+  # type: the Elasticsearch index type
+  cluster {
+    name: "elasticsearch"
+    index: "bad"
+    clusterType: "bad"
```

Comment:
Why did this get changed from `type` to `clusterType` ?

Reply:
This is related with es loader. Since we use different config library in es loader, "type" can not be used as an key because of it is one of the keyword in the Scala lang.